pangeo-notebook/conda-linux-64.lock

```diff
@@ -302,7 +301,7 @@ https://conda.anaconda.org/conda-forge/noarch/mercantile-1.1.5-pyh9f0ad1d_0.tar.
 https://conda.anaconda.org/conda-forge/noarch/munch-2.5.0-py_0.tar.bz2#31d9e9be500e25ff0050bc9f57a6bcd7
 https://conda.anaconda.org/conda-forge/linux-64/netcdf4-1.5.3-nompi_py37hec16513_103.tar.bz2#1274c6c94a897ee368fe64b1d5c41f98
 https://conda.anaconda.org/conda-forge/noarch/networkx-2.4-py_1.tar.bz2#5aaf85a2001a7d818148794f3099eb5e
-https://conda.anaconda.org/conda-forge/linux-64/numba-0.50.1-py37h0da4684_1.tar.bz2#e4896d6cdde22d25fa5cd48b31115074
+https://conda.anaconda.org/conda-forge/linux-64/numba-0.48.0-py37hb3f55d8_0.tar.bz2#2f5ebdbe82b91475079af2b431760a38
```

Comment:
as far as i can tell the only package versions that dropped by pinning datashader>=0.11 are numba